oxo.yaml

```diff
@@ -1,34 +1,34 @@
 kind: Agent
-name: semgrep
+name: opengrep
```

Comment:
Keep semgrep until we change all the occurrences after the first release of opengrep